# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/WebRenderAPI.cpp
# Commit: 5506d1803e15
# Full Hash: 5506d1803e158df5504bc74a7818cb78dca6420c
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-10-16 05:20:00
# Description:
#   Bug 1585801 - Fixing crash when folder is not writable and refactored logic. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D48999
# ==============================================================================

diff -r f24ddb55b48b -r 5506d1803e15 gfx/webrender_bindings/WebRenderAPI.cpp
--- a/gfx/webrender_bindings/WebRenderAPI.cpp	Sat Oct 12 23:50:45 2019 +0000
+++ b/gfx/webrender_bindings/WebRenderAPI.cpp	Tue Oct 15 17:03:22 2019 +0000
@@ -584,6 +584,10 @@
   wr_api_capture(mDocHandle, path, bits);
 }
 
+void WebRenderAPI::SetTransactionLogging(bool aValue) {
+  wr_api_set_transaction_logging(mDocHandle, aValue);
+}
+
 void WebRenderAPI::SetCompositionRecorder(
     UniquePtr<layers::WebRenderCompositionRecorder> aRecorder) {
   class SetCompositionRecorderEvent final : public RendererEvent {